### PR TITLE
release() warns if NEWS.Rd exists

### LINKED ISSUE #1133: release() should warn if both inst/NEWS.Rd and NEWS.md exist
Just happened to me; the former takes precedence, which is probably not what you want.

### PR DESCRIPTION
**Pull request by krlmlr**
Fixes #1133.

### CODE REVIEW

File: R/check-devtools.r [hunk 1]
```diff
@@ -98,6 +98,13 @@ check_news_md <- function(pkg) {
     "NEWS.md is not ignored",
     "NEWS.md now supported by CRAN and doesn't need to be ignored."
   )
+
+  news_rd_path <- file.path(pkg$path, "inst/NEWS.Rd")
+  check_status(
+    !file.exists(news_rd_path),
+    "NEWS.Rd removed",
```
Comment (hadley) [inline R/check-devtools.r]: "NEWS.Rd does not exist" would be a better message, I think.

Also this is going to be a false positive for people who use `inst/NEWS.Rd` only

**Comment by krlmlr** [comment]
Changed error message, added pending NEWS. This check is invoked only if NEWS.md exists.

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # devtools 1.10.0.9000
 
+* `release()` warns if both `inst/NEWS.Rd` and `NEWS.md` exist (@krlmlr, #1135).
+
+* `release()` doesn't throw error if Git head is detached (@krlmlr, #1136).
```
Comment (hadley) [inline NEWS.md]: Where did these come from? Can I ignore?

Comment (krlmlr) [inline NEWS.md]: These are good I think -- already merged (#1124, #1136) but didn't have NEWS.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**